packages/legend-application-studio/src/components/editor/editor-group/service-editor/testable/ServiceTestsEditor.tsx

```diff
@@ -147,8 +147,8 @@ export const ExternalFormatParameterEditorModal = observer(
     isReadOnly: boolean;
     onClose: () => void;
     updateParamValue: (val: string) => void;
-    bindingParamPair: {
-      binding: Binding;
+    bindingOrContentTypeParamPair: {
+      candidate: Binding | string;
```

Comment:
maybe im wrong but isnt whats important here just the content type so we can render the text in the correct format. 
Why do we need the binding. could we just not resolve the content type of the binding here and just keep it contentType, param for the ParamPairs?

Reply:
a heads up: contentType from Binding is somehow undefined in the test (only in the test)

Reply:
fixed it by loading a plugin